onnxruntime/contrib_ops/rocm/bert/fast_gelu_tunable_op.h

```diff
@@ -33,6 +33,9 @@ struct FastGeluParams : OpParams {
 
 template <typename T, int ThreadsPerBlock, int VecSize>
 Status FastGeluOp(const FastGeluParams<T>* params) {
+  // TODO(anyone): Add tail handling for FastGelu
+  TUNABLE_OP_RETURN_UNSUPPOTED_ARGUMENT_IF(!(params->bias_length > 0 && (params->bias_length % VecSize == 0)));
```

Comment:
I just realized there is a subtle difference between IsSupported and Condition: if we use IsSupported, we need to guarantee that there is at least one kernel that is supported. I think the logic should be the following to make sure we always have a supported kernel when bias_length is 0.

!((params->bias_length > 0 && params->bias_length % VecSize == 0 && params->input_length % VecSize == 0) || params->input_length % VecSize == 0)

Reply:
Yeah, that's it. And every specialized kenel implementations check the supporting status for itself, this will also help us to keep our sanity ;)